### ISSUE TITLE
Library of research notes

### ISSUE THREAD

**Issue by Lightningblade** [jackowski626/universal_tc_research_solver#3]
Have a suggestion / prediction table to be able to select from.

**Comment by jackowski626** [comment]
You mean something that suggests research notes as one places aspects on the grid? That would be fancy, even if research is sometimes changed in modpacks. If I have time I can look into this

**Comment by Lightningblade** [comment]
Yes something like that. Now I haven't looked at the code yet, but if there is a connection between the client and a server, one could cache/save the grids that the client tries to solve. 

Another more solid solution would be to run a script on the file in Thaumcraft that contains all the layouts.  I'm not sure though where they save the recipes. https://github.com/Azanor/thaumcraft-api/blob/79427c4b9b923315bfb7ad28a15254b0cb07160d/research/_example.json.txt

**Comment by jackowski626** [comment]
This would be a huge time saver for sure. Everything the solver does happens on the client
